backend/tournesol/tests/test_api_contributor_recommendations.py

```diff
@@ -333,3 +334,18 @@ def test_recommendations_are_sorted_by_descending_total_score(self):
             self.assertEqual(
                 response.data["results"][i]["uid"], score_to_entity[score].uid
             )
+
+    def test_recommendations_include_entity_no_extra_field_fields(self):
```

Comment:
This test may become obsolete if we remove the extra field added in `ContributorRecommendationSerializer`

Reply:
After some tests, you can directly check the presence of the `type` field in the test  `test_api_polls.PollsRecommendationsApi.test_anonymous_can_list_recommendations` , and remove this test.

With something like this:

```python
self.assertEqual(results[0]["type"], "video")
```